common/traits/wc_race_traits.txt

```diff
@@ -0,0 +1,1488 @@
+		# first_valid = {
+			# triggered_desc = {
+				# trigger = {
+					# NOT = { exists = this }
+				# }
+				# desc = trait_creature_moarg_desc
+			# }
+			# desc = trait_creature_moarg_character_desc
+		# }
+	# }
+# }
+creature_observer = {
+	index = 2067
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_observer_desc
+			}
+			desc = trait_creature_observer_character_desc
+		}
+	}
+}
+
+# Elementals
+# creature_air_elemental = {
+	# index = 2068
+	# desc = {
+		# first_valid = {
+			# triggered_desc = {
+				# trigger = {
+					# NOT = { exists = this }
+				# }
+				# desc = trait_creature_air_elemental_desc
+			# }
+			# desc = trait_creature_air_elemental_character_desc
+		# }
+	# }
+# }
+# creature_earth_elemental = {
+	# index = 2069
+	# desc = {
+		# first_valid = {
+			# triggered_desc = {
+				# trigger = {
+					# NOT = { exists = this }
+				# }
+				# desc = trait_creature_earth_elemental_desc
+			# }
+			# desc = trait_creature_earth_elemental_character_desc
+		# }
+	# }
+# }
+# creature_fire_elemental = {
+	# index = 2070
+	# desc = {
+		# first_valid = {
+			# triggered_desc = {
+				# trigger = {
+					# NOT = { exists = this }
+				# }
+				# desc = trait_creature_fire_elemental_desc
+			# }
+			# desc = trait_creature_fire_elemental_character_desc
+		# }
+	# }
+# }
+creature_flamewaker = {
+	index = 2071
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_flamewaker_desc
+			}
+			desc = trait_creature_flamewaker_character_desc
+		}
+	}
+}
+# creature_water_elemental = {
+	# index = 2072
+	# desc = {
+		# first_valid = {
+			# triggered_desc = {
+				# trigger = {
+					# NOT = { exists = this }
+				# }
+				# desc = trait_creature_water_elemental_desc
+			# }
+			# desc = trait_creature_water_elemental_character_desc
+		# }
+	# }
+# }
+
+creature_iron_dwarf = {
+	index = 2073
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_iron_dwarf_desc
+			}
+			desc = trait_creature_iron_dwarf_character_desc
+		}
+	}
+}
+creature_vrykul = {
+	index = 2074
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_vrykul_desc
+			}
+			desc = trait_creature_vrykul_character_desc
+		}
+	}
+}
+creature_tuskarr = {
+	index = 2075
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_tuskarr_desc
+			}
+			desc = trait_creature_tuskarr_character_desc
+		}
+	}
+}
+creature_magnataur = {
+	index = 2076
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_magnataur_desc
+			}
+			desc = trait_creature_magnataur_character_desc
+		}
+	}
+}
+creature_furbolg = {
```

Comment:
I suggest using less tech-savy furbolg as trait icon
Current one is holding a rifle on an icon

Reply:
Changed art to a more tribal looking Furbolg 460a2eaa0acff8acc5f38ca4e6188a3a7a3948f2